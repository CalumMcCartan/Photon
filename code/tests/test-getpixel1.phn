func int main()
{
  #whatever
  image img;
  
  int index;
  int p;
  int x;
  int y;
  int position;
  int channels;
  int[] ps;

  
  img = load("edwards.png");

  #for gettign a specific pixel, input what you want
  x = 1;
  y = 0;

  for( index = 0; index <  4; index = index+1 ) {
    p = get_pixel(img, x +index, y);
    array_add(ps,p);
    print(p);
    img = load("sky.jpg");
  }

  for ( index = 0; index < ps.length; index = index+1) {
    print(ps[index]);
  }

  return 0;
}