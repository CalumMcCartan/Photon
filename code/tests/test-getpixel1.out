210
228
230
255
210
228
230
255
